packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -165,7 +165,7 @@ export function loadBasedir(basedir, options = {}) {
 async function resolveSpecFromConfig(referrer, specPath) {
   await null;
   try {
-    return new URL(await resolveModuleSpecifier(specPath, referrer)).pathname;
+    return new URL(await importMetaResolve(specPath, referrer)).pathname;
```

Comment:
suggestion: `createRequire(referrer).resolve(specPath)` would avoid a 3rd-party dependency and an `await`.

Reply:
#11321